src/db/postgres/lrsql/postgres/sql/query.sql

```diff
@@ -287,3 +287,50 @@ AND secret_key = :secret-key;
 SELECT scope FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key;
+
+/* LRS Status */
+
+-- :name query-statement-count
+-- :command :query
+-- :result :one
+-- :doc Return the number of statements in the LRS
+SELECT COUNT(id) scount
+FROM xapi_statement;
+
+-- :name query-actor-count
+-- :command :query
+-- :result :one
+-- :doc Return the number of distinct statement actors
+SELECT COUNT(DISTINCT actor_ifi) acount
+FROM statement_to_actor
+WHERE usage = 'Actor';
+
+-- :name query-last-statement-stored
+-- :command :query
+-- :result :one
+-- :doc Return the stored timestamp of the most recent statement
+SELECT payload->>'stored' lstored
+FROM xapi_statement
+ORDER BY id DESC
+LIMIT 1;
+
+-- :name query-platform-frequency
+-- :command :query
+-- :result :many
+-- :doc Return counts of platforms used in statements.
+SELECT COALESCE(payload#>>'{context,platform}', 'unknown') platform,
```

Comment:
Huh apparently GitHub does not like this

Reply:
Yeah it is not uncommon for GH syntax highlighting to break. To be fair SQL syntax is almost impossible to predict across implementations and it won't get better until they delegate it to an LLM or something.